docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -46,7 +46,10 @@ After downloading, go to your downloads directory, decompress the file and add i
 ```
 **IMPORTANT NOTE**: 
 1. The **reference_genome** table needs to be populated before updating the **gene** table. Further details can be found in [this document](import-reference-genome.md). 
-2. Loading genes only to the **reference_genome_gene** table without updating the **gene** table, please use the following command:
+2. use **--species** option if you import genes from other kinds of organisms other than homo species
```

Comment:
Please update:

2. Use **--species** option when importing genes for a species other than human.

Reply:
@pieterlukasse @sandertan , thanks for your review. I will find time later today to get code modified. I totally agree with your both to have genome.name and genome.build removed from the portal.properties and using hard-code genome information in the validator for now.